index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>
+        My Tribute Page
+    </title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+</head>
+
+<body>
+    <main id="main">
+        <h1 id="title"> Dr. Norman Borlaug</h1>
+        <p> The man who saved a billion lives
+        <p>
+        <div id="img-div">
+            <img id="image" src="img1.jpg" alt="A very kind man" />
+            <figure id="img-caption">
+                <p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields
+                    - part
+                    of his life-long war on hunger.</p>
+            </figure>
+        </div>
+        <section id="tribute-info">
+            <p id="timeline"><strong>Here's a time line of Dr. Borlaug's life:</strong></p>
```

Comment:
Please use the class attribute for multiple elements and not the ID selector since the "id" selector is unique on a page and can only apply to at most one element, while the “class” selector can apply to multiple elements.